src/components/Modal/modal.scss

```diff
@@ -56,6 +56,7 @@
     bottom: 0;
     display: flex;
     flex-direction: row;
+    gap: 10px;
```

Comment:
Hi @lisilinhart since we are removing a margin of 20px, shouldn't the gap be 20px as well?

Reply:
No, Nikola doesn't want the gap to be so big